test/expect/TestOperators.test_addmm.expect

```diff
@@ -4,10 +4,20 @@ producer_version: "0.2"
 domain: "com.facebook"
 graph {
   node {
+    input: "3"
+    output: "5"
+    op_type: "expand"
```

Comment:
This does not look good.

Reply:
Yeah, the `expand` is the broadcasting stuff which I am working on fixing.